Sources/SuperDelegate+OpenURL.swift

```diff
@@ -28,13 +28,11 @@ import Foundation
 public protocol OpenURLCapable: ApplicationLaunched {
     /// Called when your application has been launched due to a URL.
     /// @return Whether the URL can be handled from a cold start.
-    @warn_unused_result
-    func canOpenLaunchURL(launchURLToOpen: URLToOpen) -> Bool
+    func canOpen(launchURL: URLToOpen) -> Bool
```

Comment:
well I couldn't call it `URL`. It is a URL that I'm passing you with other data and you should open it. Really happy to hear naming suggestions here... was never thrilled with it.